examples/twitter-nlp/requirements.txt

```diff
@@ -0,0 +1,4 @@
+click==8.0.1
+colorama==0.4.4
```

Comment:
colorama doesn't look to have been updated since Oct 2020 https://pypi.org/project/colorama/#history
could we consider rich here? https://pypi.org/project/rich/

Reply:
yep, updating to rich